isthmus/src/main/java/io/substrait/isthmus/expression/FunctionConverter.java

```diff
@@ -69,9 +73,48 @@ public FunctionConverter(
       }
     }
 
+    for (var entry : alm.entries()) {
+      String key = entry.getKey();
+      var func = entry.getValue();
+      for (FunctionMappings.Sig sig : calciteOperators.get(key)) {
+        substraitFuncKeyToSqlOperatorMap.put(func.key(), sig.operator());
+      }
+    }
+
     this.signatures = matcherMap;
   }
 
+  protected Optional<SqlOperator> getSqlOperatorFromSubstraitFunc(
+      Expression.ScalarFunctionInvocation expr) {
+    var resolver = getTypeBasedResolver();
+    if (substraitFuncKeyToSqlOperatorMap.containsKey(expr.declaration().key())) {
```

Comment:
(style/readability) better to invert this condition and inner clause so we don't indent a large block.

Reply:
changed.